[TH2-3602] Add downloading logging config from k8s (#197)
th2-net/th2-common-j: README.md

## Release notes

### 3.37.1

+ Fixed:

>   + When creating the `CommonFactory` from k8s the logging configuration is downloaded as well

Reviewer: ```suggestion
  + When creating the `CommonFactory` from k8s the logging configuration wouldn't be downloaded
```
